src/components/Main/Post.js

```diff
@@ -0,0 +1,91 @@
+const likeIcon = `<svg viewBox="0 0 24 24" aria-hidden="true" fill="#757575"><g><path d="M16.697 5.5c-1.222-.06-2.679.51-3.89 2.16l-.805 1.09-.806-1.09C9.984 6.01 8.526 5.44 7.304 5.5c-1.243.07-2.349.78-2.91 1.91-.552 1.12-.633 2.78.479 4.82 1.074 1.97 3.257 4.27 7.129 6.61 3.87-2.34 6.052-4.64 7.126-6.61 1.111-2.04 1.03-3.7.477-4.82-.561-1.13-1.666-1.84-2.908-1.91zm4.187 7.69c-1.351 2.48-4.001 5.12-8.379 7.67l-.503.3-.504-.3c-4.379-2.55-7.029-5.19-8.382-7.67-1.36-2.5-1.41-4.86-.514-6.67.887-1.79 2.647-2.91 4.601-3.01 1.651-.09 3.368.56 4.798 2.01 1.429-1.45 3.146-2.1 4.796-2.01 1.954.1 3.714 1.22 4.601 3.01.896 1.81.846 4.17-.514 6.67z"></path></g></svg>`;
+const viewIcon = `<svg viewBox="0 0 24 24" aria-hidden="true" fill="#757575"><g><path d="M8.75 21V3h2v18h-2zM18 21V8.5h2V21h-2zM4 21l.004-10h2L6 21H4zm9.248 0v-7h2v7h-2z"></path></g></svg>`;
+const bookmarkIcon = `<svg viewBox="0 0 24 24" aria-hidden="true" fill="#757575"><g><path d="M4 4.5C4 3.12 5.119 2 6.5 2h11C18.881 2 20 3.12 20 4.5v18.44l-8-5.71-8 5.71V4.5zM6.5 4c-.276 0-.5.22-.5.5v14.56l6-4.29 6 4.29V4.5c0-.28-.224-.5-.5-.5h-11z"></path></g></svg>`;
+const shareIcon = `<svg viewBox="0 0 24 24" aria-hidden="true" fill="#757575"><g><path d="M12 2.59l5.7 5.7-1.41 1.42L13 6.41V16h-2V6.41l-3.3 3.3-1.41-1.42L12 2.59zM21 15l-.02 3.51c0 1.38-1.12 2.49-2.5 2.49H5.5C4.11 21 3 19.88 3 18.5V15h2v3.5c0 .28.22.5.5.5h12.98c.28 0 .5-.22.5-.5L19 15h2z"></path></g></svg>`;
+
+function Post({
+  pinned = false,
+  content,
+  replies,
+  reposts,
+  likes,
+  views,
+  date,
+  id,
+  profileExtention,
+  name,
+  verified = false,
+}) {
+  const profilePicture = require(`./../../assets/img/accounts/${id}/profile.${profileExtention}`);
+  return (
+    <div className="post-container">
+      <div className="line"></div>
+      <div className="post">
+        {pinned && (
+          <div className="pinned">
+            <strong>
+              <span dangerouslySetInnerHTML={{ __html: pinnedIcon }}></span>
+              <p>Pinned</p>
+            </strong>
+          </div>
+        )}
+        <div className="body">
+          <div className="img">
+            <img src={profilePicture} alt="profile" />
+          </div>
+          <div className="post-info">
+            <div className="header">
+              <div>
+                <p>{name}</p>
+                {verified && (
+                  <img src={verifiedIcon} alt="verifed" className="verified" />
+                )}
+                <div className="secondary-coloring-div">
+                  <p>@{id}</p>
+                  <p>&nbsp;</p>
+                  <p>{date}</p>
+                </div>
+              </div>
+              <div className="three-dots">...</div>
+            </div>
+            <div className="content">
+              <p dangerouslySetInnerHTML={{ __html: content }}></p>
```

Comment:
I'm escaping the content when take it, so I can prevent errors, here is an example:
the content may contain a link, which is representer in <a></a>, if I directly put the content inside the <p></p> it will display the text "<a></a>" instead of a link, that's why